docs/modules/ROOT/pages/explanations/cluster_monitoring.adoc

```diff
@@ -31,20 +31,24 @@ The result of doing everything again, would be a huge waste of resources both in
 
 For that reason we'll make use of Cluster Monitoring as much as possible.
 
-To be able to change the existing alerting rules we'll bring our own set of rules based on the https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator] (which in turn is based on https://github.com/prometheus-operator/kube-prometheus[kube-prometheus]).
+To be able to change the existing alerting rules bring our own set of rules based on the https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator] (which in turn is based on https://github.com/prometheus-operator/kube-prometheus[kube-prometheus]).
 These rules we can change and adapt to our liking with Jsonnet.
 The current rules are ignored by creating https://prometheus.io/docs/alerting/latest/alertmanager/#silences[silences] in AlertManager.
 
-Additionally we'll use the https://docs.openshift.com/container-platform/latest/monitoring/monitoring-your-own-services.html[user workload monitoring] feature to add new scrape targets and Prometheus alert rules.
+Additionally we use the https://docs.openshift.com/container-platform/latest/monitoring/monitoring-your-own-services.html[user workload monitoring] feature to add new scrape targets and Prometheus alert rules.
```

Comment:
Well, that was the idea. If this works or not, I can not tell. At least, we have not testet this yet.